src/main/java/com/microsoft/azure/relay/ClientWebSocket.java

```diff
@@ -475,4 +478,20 @@ private CompletableFuture<Void> readFragmentsAsync() {
 					});
 		}
 	}
+	
+	private class PingRunnable implements Runnable {
```

Comment:
Would there be any consideration to split this private class into another file vs keeping it here? and are there any advantages/disadvantages of doing it one way?

Reply:
I think it makes sense to keep it nested since this functionality is not used anywhere else (and there are no plans for this) and it uses some of ClientWebSocket's private variables.